2023/ranehegbe/README.md

```diff
@@ -5,3 +5,89 @@ In my free time, I like to play the piano, watch romcoms, and read.
+- [ ] Write a short blog post which demonstrates your expertise in technical writing. In English, with at least one image, less than 500 words. Demonstrate your expertise based on past experiences. For example, any specific documentation writing you’ve done in the past, or other technical writing you’ve made. The more specific the better. The post has to be posted on a publicly-available platform ([Dev.to](https://dev.to/), [Hashnode](https://hashnode.com/), [Medium](https://medium.com/), your own website),
+- [ ] Ask @thibaudcolas via email or on Slack to give you editor access to a Wagtail website. Once you’ve received access, we ask you to create a new page demonstrating some of Wagtail’s accessibility features.
+- [ ] With your access to the demo site and by reviewing our user guide, identify a feature of Wagtail which isn’t documented, and report it back on the guide project issue tracker. Make sure to explain what the feature is, why and how you think it should be documented.
+- [ ] Draft the new content to document the feature of Wagtail which you’ve reported isn’t documented. Add it to the issue you created as a new comment, so we can provide feedback.
+- [ ] Ask @thibaudcolas via email or on Slack to give you editor access to [guide.wagtail.org](https://guide.wagtail.org). Once you’ve received access, we ask you to make the necessary changes in the CMS to add the feature documentation.
+- [ ] TBC – Additional tasks specific to the Outreachy application process.
+
+## Research
+### Understanding Wagtail
+- [Wagtail Website](https://wagtail.org/)
+- [Wagtail Documentation - Getting Started](https://docs.wagtail.org/en/latest/getting_started/index.html)
+- [Wagtail Documentation - The Zen of Wagtail](https://docs.wagtail.org/en/latest/getting_started/the_zen_of_wagtail.html)
+- [Wagtail's Youtube Channel](https://www.youtube.com/@wagtail4333)
+
+### Understanding Accessibility
+- [World Wide Web Consortium(W3C) Accessibility Section](https://www.w3.org/mission/accessibility/)
+- [Silktide's Youtube videos on Accessibilty](https://youtu.be/5H1JGdqLrWo?si=al2bQUmmohGD2XvH)
+- [Wagtail.org's accessibility page](https://wagtail.org/accessibility/)
+- [Sa11y](https://sa11y.netlify.app/accessibility/)
+- [ATAG Guidelines](https://www.w3.org/TR/ATAG20/)
+- [WCAG Guidelines](https://www.w3.org/TR/WCAG22/)
+- [Wagtail Accessibility Team's Github Page](https://github.com/wagtail/wagtail/wiki/Accessibility-team)
+
+### Things to know about technical documentation
+- [How to create technical documentation](https://slite.com/learn/technical-documentation)
+- [5 steps to create technical documentation that's (actually) helpful](https://plan.io/blog/technical-documentation/)
+
+## Blog post showing my interest in learning
+Click [Here](https://medium.com/@ranehobasi/preserving-heritage-embracing-esan-a-nigerian-language-in-the-digital-era-d5b8a4ce5217)
```

Comment:
- "Click here" is a really bad pattern for accessibility :/
- Your blog post is 511 words excluding the title, above the 500 word limit I had set.